sdcm/sct_config.py

```diff
@@ -569,13 +571,13 @@ class SCTConfiguration(dict):
         dict(name="ami_id_db_oracle", env="SCT_AMI_ID_DB_ORACLE", type=str,
              help="AMS AMI id to use for oracle node"),
 
-        dict(name="aws_root_disk_size_db", env="SCT_AWS_ROOT_DISK_SIZE_DB", type=int,
```

Comment:
I would suggest separating this to commit on it's own, since lots of configuration yamls also need to be changed